modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/outputs/EnrichedEventSpec.scala

```diff
@@ -112,42 +112,42 @@ class EnrichedEventSpec extends Specification {
       testField(_.ti_name = "ti_name", _.ti_name)
       testField(_.ti_category = "ti_category", _.ti_category)
       testField(_.ti_price = "ti_price", _.ti_price)
-      testField(_.ti_quantity = new JInteger(0), _.ti_quantity)
-      testField(_.pp_xoffset_min = new JInteger(0), _.pp_xoffset_min)
-      testField(_.pp_xoffset_max = new JInteger(0), _.pp_xoffset_max)
-      testField(_.pp_yoffset_min = new JInteger(0), _.pp_yoffset_min)
-      testField(_.pp_yoffset_max = new JInteger(0), _.pp_yoffset_max)
+      testField(_.ti_quantity = JInteger.valueOf(0), _.ti_quantity)
+      testField(_.pp_xoffset_min = JInteger.valueOf(0), _.pp_xoffset_min)
+      testField(_.pp_xoffset_max = JInteger.valueOf(0), _.pp_xoffset_max)
+      testField(_.pp_yoffset_min = JInteger.valueOf(0), _.pp_yoffset_min)
+      testField(_.pp_yoffset_max = JInteger.valueOf(0), _.pp_yoffset_max)
```

Comment:
All of these were taken care of in https://github.com/snowplow/enrich/commit/86536a539e4b5dd659e1d9d07bd76f69e3bef454

I should have created a PR just for this and merge right away. I didn't because it was preventing me from compiling but actually I could have created a new branch just for this and rebase my branch on top of it waiting for the merge. I will do that next time. Sorry!

Reply:
No worries @benjben ! Thanks for the empathy!